### ISSUE TITLE
Sanitization issue

### ISSUE THREAD

**Issue by cristinnaiulia** [WPChill/strong-testimonials#236]
Please be aware, you have some sanitization issues in your plugin
strong-testimonials/admin/custom-fields-ajax.php:44: $new_field_type = $_REQUEST['fieldType'];
strong-testimonials/admin/custom-fields-ajax.php:45: $new_field_class = $_REQUEST['fieldClass'];
strong-testimonials/admin/class-strong-testimonials-admin-scripts.php:290: $tab = isset( $_GET['tab'] ) ? $_GET['tab'] : '';
strong-testimonials/admin/class-strong-testimonials-admin-scripts.php:319: $tab = isset( $_GET['tab'] ) ? $_GET['tab'] : '';
strong-testimonials/admin/custom-fields.php:35: $form_id = $_POST['form_id'];
And more.
Please take some time to give your whole plugin a look through about those.

**Comment by giucu91** [comment]
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L548
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L585
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L587 -> incearca wp_kses_post
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L624-L626 -> incearca wp_kses_post
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L721 -> nu avem voie cu traduceri de variabile
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L715
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L713 -> la id trebuie esc_attr la name nu are nimic
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L736
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L778-L779
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L868 esc_attr e pentru atribute, label nu e atribut, trebuie esc_html
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L898
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L945
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L967 value si label
https://github.com/WPChill/strong-testimonials/blob/dev/admin/class-strong-testimonials-helper.php#L996
